src/System.Memory/src/System.Memory.csproj

```diff
@@ -8,6 +8,9 @@
     <Configurations>netcoreapp-Unix-Debug;netcoreapp-Unix-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netcoreappaot-Windows_NT-Debug;netcoreappaot-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="System\Buffers\SequenceReaderExtensions_binary.cs" />
+    <Compile Include="System\Buffers\SequenceReader.cs" />
+    <Compile Include="System\Buffers\SequenceReader_search.cs" />
```

Comment:
Use '.' instead of underscore?

Reply:
Matching existing naming.

Reply:
In almost all other places here we use '.'. I know we aren't consistent (1-2 ReadOnlySequence related files have underscore), but I just changed to use '.' in my JsonReader PR. Also, all the Utf8Parser files use '.'